verilog/analysis/checkers/proper_parameter_declaration_rule.cc

```diff
@@ -107,9 +112,37 @@ static const Matcher &ParamDeclMatcher() {
   return matcher;
 }
 
+void ProperParameterDeclarationRule::AddParameterViolation(
+    const verible::Symbol &symbol, const verible::SyntaxTreeContext &context) {
+  auto *token = GetParameterToken(symbol);
```

Comment:
try to avoid `auto` if the type is not so complicated that it would make the line unnecessary long.

```
const verible::TokenInfo *token = ...
```